modules/security/src/main/java/io/oasp/module/security/common/base/accesscontrol/AbstractAccessControlBasedAuthenticationProvider.java

```diff
@@ -148,12 +87,12 @@ protected UserDetails retrieveUser(String username, UsernamePasswordAuthenticati
     for (AccessControl accessControl : accessControlSet) {
       authorities.add(new AccessControlGrantedAuthority(accessControl));
     }
-
-    String password = null;
-    Object credentials = authentication.getCredentials();
-    if (credentials != null) {
-      password = credentials.toString();
-    }
-    return createUser(username, password, principal, authorities);
+    return authorities;
   }
+
+  /**
+   * @param username
```

Comment:
hum that is strange, the current version has all TODOS removed and JavaDocs added. Your comments say

>    hohwille commented on an outdated diff